panel/widgets/examples.py

```diff
@@ -0,0 +1,295 @@
+import os
```

Comment:
- The Gradio [`Examples`](https://www.gradio.app/docs/gradio/examples) has another api and many more (advanced) features like *batch inference* and *example caching*. Should we align closer with that api?